<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateSubjectWindow</class>
 <widget class="QWidget" name="CreateSubjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>题目详情编辑</string>
  </property>
  <widget class="QLineEdit" name="majorLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>8</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>9</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>科目名：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveMajor">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addTask">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加题目</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addCategory">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加分类</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentMajorName">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>科目名：</string>
   </property>
  </widget>
  <widget class="QTableView" name="categoryView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>181</width>
     <height>251</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QTableView" name="taskView">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>511</width>
     <height>251</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
